### PR TITLE
Feature/reactjs

### PR DESCRIPTION
**Pull request by illia108**

### CODE REVIEW

File: app/controllers/api/v1/favourites_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class Api::V1::FavouritesController < ApplicationController
```
Comment (BogdanLinnik) [inline app/controllers/api/v1/favourites_controller.rb]: why v1?

File: app/javascript/favourites/components/Remove.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+class Remove extends React.Component {
+  constructor() {
+    super();
+  }
```
Comment (BogdanLinnik) [inline app/javascript/favourites/components/Remove.jsx]: Did we need constructor if there is no state?

File: config/routes.rb [hunk 3]
```diff
@@ -17,11 +17,24 @@
...
       resources :ratings, only: [:create, :update, :destroy]
     end
+
+    resources :users do
+      resources :favourites
+    end
+  end
+
+  namespace :api, defaults: { format: :json } do
+    namespace :v1 do
```
Comment (BogdanLinnik) [inline config/routes.rb]: Same

File: app/controllers/api/v1/favourites_controller.rb (hunk 1 above)
Comment (illia108) [reply app/controllers/api/v1/favourites_controller.rb]: It's a good practice to use versioning for API. If we decide to modify our API, we create a new version. So everyone/everything that is dependent on our old API will still work.

File: app/javascript/favourites/components/Remove.jsx (hunk 2 above)
Comment (illia108) [reply app/javascript/favourites/components/Remove.jsx]: No. We don't. I'll remove it. Thanks

File: config/routes.rb (hunk 3 above)
Comment (illia108) [reply config/routes.rb]: Was explained above.

**Pull request merged**